### PR TITLE
Handled dot paths in ScrollToErrorField component

### LINKED ISSUE #1741: Handle nested formik values (dot paths) in ScrollToErrorField component
Ref: https://bigbinary.slack.com/archives/C03T5R6QYSC/p1688026200465819?thread_ts=1688019939.823089&cid=C03T5R6QYSC

@soorajjtk _a

### PR DESCRIPTION
**Pull request by Calvinwilson99**
- Fixes #1741 

**Description**

- Fixed: Handled dot paths in _ScrollToErrorField_.

**Checklist**

- [x] I have made corresponding changes to the documentation.
- ~~[ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

### CODE REVIEW

**Comment by Calvinwilson99** [comment]
@ajmaln _a. Please review.

**Review by ajmaln — COMMENTED** [review]
@Calvinwilson99 _a Let's add a story as discussed

**Comment by Calvinwilson99** [comment]
@ajmaln _a. Added a story showing scrolling in action.

**Review by ajmaln — APPROVED** [review]
@amaldinesh7 _a Please review

**Review by amaldinesh7 — APPROVED** [review]

File: src/components/formik/Form/index.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import FormWrapper from "./FormWrapper";
 
 const Form = forwardRef(
   (
-    { className, children, formikProps, formProps, scrollToErrorField = false },
+    { className, children, formikProps, formProps, scrollToErrorField = true },
```
Comment (amaldinesh7) [inline src/components/formik/Form/index.jsx:10]: We can keep the default value as false itself.

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@Calvinwilson99 _a Please check the comment.

Update the description as per the format.

**Comment by Calvinwilson99** [comment]
@amaldinesh7 _a. Made the requested changes.

**Pull request merged**